graphql_api/types/user/user.py

```diff
@@ -39,3 +39,9 @@ def resolve_student_created_at(user: Owner, info) -> Optional[datetime]:
 @user_bindable.field("studentUpdatedAt")
 def resolve_student_updated_at(user: Owner, info) -> Optional[datetime]:
     return user.student_updated_at
+
+
+@user_bindable.field("customerIntent")
+def resolve_customer_intent(user: Owner, info) -> Optional[datetime]:
+    owner = user
+    return owner.user.customer_intent
```

Comment:
Wait this is odd, why is this file called user.py, all users are `Owners`, the `me` resolver creates a custom `User` type which is certain fields from the Owner object? Pretty odd, probably some legacy thing we want wanted to differentiate different use cases for it.

Do we not have a type that properly maps to the current_user/`User` class?

Reply:
I thought it was odd as well. We changed our owner/user model a while back so I'm not sure if that is related.

Reply:
Mmmm, don't want to bloat your PR but I think there's a legit use case to have a User resolver. We can do it now or later, I'll leave that up to you 👌

Reply:
Created an issue here
https://github.com/codecov/engineering-team/issues/1220